types/nanoid-dictionary/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for nanoid-dictionary 3.0
+// Project: https://github.com/CyberAP/nanoid-dictionary#readme
+// Definitions by: Shengjie Pan <https://github.com/kenelm007>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* Lowercase English letters */
+export const lowercase: string;
+/* Uppercase English letters */
+export const uppercase: string;
+/* Numbers from 0 to 9 */
+export const numbers: string;
+/* Numbers and english alphabet without lookalikes */
+export const nolookalikes: string;
```

Comment:
If you amend your changes that way:
```suggestion
export const nolookalikes: string;

/* tslint:disable:no-declare-current-package */

declare module 'nanoid-dictionary/lowercase' {
    export = lowercase;
}

declare module 'nanoid-dictionary/uppercase' {
    export = uppercase;
}

declare module 'nanoid-dictionary/numbers' {
    export = numbers;
}

declare module 'nanoid-dictionary/nolookalikes' {
    export = nolookalikes;
}
```

you will allow to use specific dictionary only, as per api:
https://github.com/CyberAP/nanoid-dictionary#usage